bin/simple-local-server.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+brew services start elasticsearch
+brew services start redis
+brew services start mysql
+
+yq -yi '.development.host="127.0.0.1" | .test.host="127.0.0.1"' config/database.yml
+sed -i -e 's/redis:6379/localhost:6379/' config/initializers/redis.rb
+sed -i -e '/TaxonLineage/ d' lib/tasks/create_elasticsearch_indices.rake
```

Comment:
Are you just trying saving time/space?
This creates a `.rake-e`... was that on purpose?

Reply:
Yep - this is to avoid the very expensive TaxonLineage indexing. I'll see if I can get rid of a .rake-e file.

Reply:
It seems like it is taking -e as the optional extension to -i option.